builtin/game/mapgen.lua

```diff
@@ -0,0 +1,12 @@
+function core.get_mapgen_bounds(mapgen_limit, chunksize)
```

Comment:
Might it be possible to use the values from `MapgenParams::calcMapgenEdges()` instead of re-implementing the functionality in Lua?

Reply:
Oh, I didn't realize that function existed, lol. That should simplify things.

Reply:
Reimplemented with this in mind.